### ISSUE TITLE
Make sure ie is js.coffee and not just coffee

### ISSUE THREAD

**Issue by maxbeizer** [centresource/preseason#58]
the JS for the ie contigency should be of the extension `.js.coffee`

**Comment by rianrainey** [comment]
@maxbeizer This may be a gap in my knowledge but why does it need to be `ie.js.coffee`? Doesn't `ie.coffee` work also?

**Comment by bval** [comment]
I agree with @rianrainey here. FWIW in Sprockets each extension you append implies another preprocessor the asset should be run through, so if there's no js preprocessor (there's not) I'm not sure what the value is in having `.js` in the filename...

Same goes for `.css.scss`.

**Comment by rianrainey** [comment]
Thanks for the clarification @bval.

I already started on this, so I'll keep it just in case...
If we want to change it, just need to:
- [ ] rename this [file](https://github.com/centresource/preseason/blob/master/lib/preseason/templates/ie8/app/assets/javascripts/ie8.coffee).
- [ ] update the path in this [file](https://github.com/centresource/preseason/blob/master/lib/preseason/recipe/ie8.rb#L12)
